.github/workflows/CD.yml

```diff
@@ -343,6 +343,7 @@ jobs:
           docker tag $IMAGE_NAME public.ecr.aws/$ECR_REPO:latest 
           docker push public.ecr.aws/$ECR_REPO:$TAG
           docker push public.ecr.aws/$ECR_REPO:latest
+          go run tools/release/adot-operator-images-mirror
```

Comment:
Since you change is extended from aws-otel-collector release workflow in a separate repo. How are you going to do the operator release? will it override/duplicate the other releases on the collector? (eg, ECR, Dockerhub, S3)

Reply:
The Operator release will just mirror the related images from Quay and GCR and push them to two ECR registries - `adot-operator` and `mirror-kube-rbac-proxy`. It won't affect the Collector release.